phpunit/functional/AutoloadTest.php

```diff
@@ -83,15 +85,26 @@ public function testAutoloadGlpiEvent()
     }
 
     #[RunInSeparateProcess]
-    public function testPluginAutoloading()
+    #[DataProvider('provideClassesAutoload')]
+    public function testPluginAutoloading(string $class): void
```

Comment:
Changes on this class are only here to separate 4 assertions into 4 tests with more readable/debuggable output, and to separate all of them in case one fails.

But overall, works exactly the same, and allowed me to add one more test.